go/tools/bzltestutil/wrap.go

```diff
@@ -126,6 +128,17 @@ func Wrap(pkg string) error {
 	cmd.Env = append(os.Environ(), "GO_TEST_WRAP=0")
 	cmd.Stderr = io.MultiWriter(os.Stderr, streamMerger.ErrW)
 	cmd.Stdout = io.MultiWriter(os.Stdout, streamMerger.OutW)
+	go func() {
```

Comment:
optional: we could abstract this to a function:
```
go signalForward(syscall.SIGTERM)
```
but maybe this would hurt readibility